spec/fixtures/pivotal_tracker_cassettes/project.yml

```diff
@@ -286,32 +229,167 @@ http_interactions:
+        stories search result resource","description":"Provide attr_accessors and
+        initialize method based on existing models.","current_state":"unstarted","requested_by_id":2415615,"url":"https://www.pivotaltracker.com/story/show/138816577","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138816529,"created_at":"2017-02-01T13:37:24Z","updated_at":"2017-02-01T18:23:14Z","estimate":1,"story_type":"feature","name":"Add
+        source commit resource","description":"Provide attr_accessors and initialize
+        method based on existing models.","current_state":"unstarted","requested_by_id":2415615,"url":"https://www.pivotaltracker.com/story/show/138816529","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138816429,"created_at":"2017-02-01T13:35:40Z","updated_at":"2017-02-01T18:23:11Z","estimate":1,"story_type":"feature","name":"Add
+        search result container resource","description":"Provide attr_accessors and
+        initialize method based on existing models.","current_state":"unstarted","requested_by_id":2415615,"url":"https://www.pivotaltracker.com/story/show/138816429","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138816401,"created_at":"2017-02-01T13:35:15Z","updated_at":"2017-02-01T18:23:08Z","estimate":1,"story_type":"feature","name":"Add
+        saved search resource","description":"Provide attr_accessors and initialize
+        method based on existing models.","current_state":"unstarted","requested_by_id":2415615,"url":"https://www.pivotaltracker.com/story/show/138816401","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138816383,"created_at":"2017-02-01T13:34:54Z","updated_at":"2017-02-01T18:23:03Z","estimate":1,"story_type":"feature","name":"Add
+        release resource","description":"Provide attr_accessors and initialize method
+        based on existing models.","current_state":"unstarted","requested_by_id":2415615,"url":"https://www.pivotaltracker.com/story/show/138816383","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138828543,"created_at":"2017-02-01T15:44:07Z","updated_at":"2017-02-01T18:22:51Z","estimate":1,"story_type":"feature","name":"Add
+        lighthouse_integration","description":"Provide attr_accessors and initialize
+        method based on existing models.","current_state":"unstarted","requested_by_id":2413445,"url":"https://www.pivotaltracker.com/story/show/138828543","project_id":<PROJECT_ID>,"owner_ids":[2413445],"labels":[],"owned_by_id":2413445},{"kind":"story","id":138755539,"created_at":"2017-01-31T21:02:55Z","updated_at":"2017-01-31T21:02:55Z","story_type":"feature","name":"Add
```

Comment:
@jasonnoble I was having some authentication issues. How should I go about hiding these?